reference/annotation/acl.rst

```diff
@@ -0,0 +1,96 @@
+@Acl
+====
+
+The ``@Acl`` annotation is used to create a new access control list and to protect the controller
+that is annotation with this ACL accordingly:
+
+.. code-block:: php
+    :linenos:
+
+    /**
+     * @Acl(
+     *     id="user_user_view",
+     *     type="entity",
+     *     class="OroUserBundle:User",
+     *     permission="VIEW"
+     * )
+     */
+    public function demoAction()
+    {
+        // ...
+    }
+
+Options
+-------
+
+``class``
+~~~~~~~~~
+
+**type**: ``string``
+
+When the `type`_ option is set to ``entity``, the fully qualified class name configured with the
```

Comment:
When the `type`_ option is set to ``entity``, the fully qualified class name configured with the
``class`` option is used to decide whether or not the ACL has to be evaluated when checking if a
user has access to a certain class. If given action have ParamConverter (http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/converters.html) parameter, and class of this parameter is the same as class parameter from the ACL annotation,  the check will be done on object level (if user have access to given object).